contracts/src/TeleporterTokenDestination.sol

```diff
@@ -179,18 +190,15 @@ abstract contract TeleporterTokenDestination is
      * to another destination bridge instance.
      * Requirements:
      *
-     * - `input.destinationBridgeAddress` cannot be the zero address
-     * - `input.recipient` cannot be the zero address
-     * - `amount` must be greater than 0
-     * - `amount` must be greater than `input.primaryFee`
+     * - {input.destinationBridgeAddress} cannot be the zero address
+     * - {input.recipient} cannot be the zero address
+     * - {amount} must be greater than 0
+     * - {amount} must be greater than {input.primaryFee}
      */
     function _send(SendTokensInput calldata input, uint256 amount) internal sendNonReentrant {
         require(input.recipient != address(0), "TeleporterTokenDestination: zero recipient address");
         require(input.requiredGasLimit > 0, "TeleporterTokenDestination: zero required gas limit");
-        require(
-            input.fallbackRecipient != address(0),
-            "TeleporterTokenDestination: zero fallback recipient address"
-        );
```

Comment:
Was it a bug that this was checked unconditionally, rather than only for the multihop case? Do we have a unit test to ensure that a zero fallback address is valid for the single hop case?

Reply:
Yeah, the value is not used in the single hop case, so it doesn't make sense to require that its not 0 there. Per @minghinmatthewlam's other suggestion, I've now added a requirement that it is the 0 address is in cases where it is not used, so all success case unit tests are covering that case.